### ISSUE TITLE
Improve highlight color contrast

### ISSUE THREAD

**Issue by JRegimbal** [DDMAL/Neon#582]
The yellow highlight doesn't have much contrast with the background image in many situations and often makes it harder to edit rather than easier. It should be replaced with another color that has high enough contrast with the other highlight colors, or simply removed.

**Comment by JRegimbal** [comment]
Guidelines for [non-text contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) may prove useful to anyone doing something similar.

**Comment by JRegimbal** [comment]
None of the highlight colors except the dark blue (`#0072B2`) meet WCAG 2.1 level AA requirements for UI components (3:1 contrast) against a beige (`#E2DBD2`) background typical as a manuscript background.

**Comment by JRegimbal** [comment]
One possible (albeit less rainbow) could be `#a76caa`, `#6c65a3`, `#5360a5`, `#9a3539`, and `#843234`. The issue is many colors (orange, yellow, green, light blue) can't reach the 3.0 contrast with the beige background to meet WCAG standards without also ending up with too similar a contrast to another highlight color, making them indistinguishable in greyscale.
